sensor_msgs/PointCloud2 point_cloud
tangible/BoundingBox bounding_box